tests/vmidefaults_test.go

```diff
@@ -220,4 +221,53 @@ var _ = Describe("[Serial][sig-compute]VMIDefaults", func() {
 
 	})
 
+	Context("Input defaults", func() {
+
+		It("[test_id:TODO]Should be set in VirtualMachineInstance", func() {
+
+			By("Creating a VirtualMachineInstance with an input device without a bus or type set")
+			vmi = tests.NewRandomVMI()
```

Comment:
@davidvossel Is NewRandomVMI() the preferred way, or would be using a VM instead of a VMI, better here?

Reply:
I'll switch to libvmi.